### PR TITLE
Merge LDAP changes

### PR DESCRIPTION
**Pull request by griffingarland**
I made a number of changes to solace samples for the LDAP feature.

To be merged on or shortly after July 6th (ie once the tile is released).

### CODE REVIEW

File: docs/_tutorials/ldap.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+layout: tutorials
+title: Configuring LDAP
+summary: How to setup an LDAP Server to work with Solace and some examples on how LDAP auth works
+icon: ldap.png
+---
+
+## Overview
+
+LDAP is directory-based application protocol, which Solace uses for user authentication and authorization.
```
Comment (jschabowsky) [inline docs/_tutorials/ldap.md]: which Solace "can use"... We have other options such as internal.

File: docs/_tutorials/ldap.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+Where bindDNUser and binDNPassword are the bind DN credentials (if configured).
+
+## Application Access Setup 
+
+You need to setup LDAP in the Solace tile correctly, see the [Solace Messaging Documentation]({{ site.links-ldap-settings }}).
+
+In this case, the the username is `hank` and the password is `hunter2`.
+
+Set two environment variables called `LDAP_CLIENTUSERNAME` and `LDAP_CLIENTPASSWORD` with the credentials.
```
Comment (jschabowsky) [inline docs/_tutorials/ldap.md]: This is an operating system environment variable you are setting? May want to make that clear... Also wouldnt that be a major security issue by having the password in plain text?

File: docs/_tutorials/ldap.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+## Assumptions
+
+This tutorial assumes the following:
+
+* You are familiar with Solace [core concepts]({{ site.docs-core-concepts }}){:target="_top"}.
+* You are familiar with [Cloud Foundry](https://www.cloudfoundry.org/){:target="_blank"}.
+* You have access to a running Pivotal Cloud Foundry environment.
+* Solace Messaging for PCF has been installed in your Pivotal Cloud Foundry environment.
+* You have completed the Java app tutorial.
+* You are using OpenLDAP. Some things may be different if you are using a different LDAP implementation.
```
Comment (jschabowsky) [inline docs/_tutorials/ldap.md]: You might want to add: How groups within LDAP can be mapped to VMR permissions (or something like that).

**Comment by griffingarland** [comment]
> You might want to add: How groups within LDAP can be mapped to VMR permissions (or something like that).

I tried to redo that in the introduction:

> On the VMR, groups are configured to be mapped to certain levels of authorization, and users are authorized based on the groups they belong to.

Let me know if that's clear enough.

**Comment by jschabowsky** [comment]
I am good with that.

**Comment by griffingarland** [comment]
Philippe also clarified with me that it using cf environment variables, not OS environment variables. So I changed it to reflect that.

**Comment by jschabowsky** [comment]
That makes more sense. 😉

**Comment by jschabowsky** [comment]
All looks good!

**Pull request merged**